azure/blobs_logs_monitoring/index.js

```diff
@@ -4,7 +4,7 @@
 // Copyright 2021 Datadog, Inc.
```

Comment:
nit: update Copyright year?

Reply:
I'll update. The copyright on the other forwarder is also likely old